README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# A Short Introduction of Myself
+Hello, this is **Vivian Lei**.
+I am a second-year student at the college, majoring in *English Language & Literature* and *Sociology*. 
+I am from Yunnan Province, China, but I currently live in Chicago.
```

Comment:
Yunnan is such a beautiful place, I (a Hunanese) lived there for a month a few years ago. Miss it so much!